tethys_cli/manage_commands.py

```diff
@@ -8,22 +8,25 @@
 ********************************************************************************
 """
 
-from tethys_cli.cli_helpers import get_manage_path, run_process
+from tethys_cli.cli_helpers import get_manage_path, get_django_manage_commands, run_process
 
 MANAGE_START = 'start'
 MANAGE_COLLECTSTATIC = 'collectstatic'
 MANAGE_COLLECTWORKSPACES = 'collectworkspaces'
 MANAGE_COLLECT = 'collectall'
 MANAGE_CREATESUPERUSER = 'createsuperuser'
-MANAGE_SHELL = 'shell'
+MANAGE_GET_PATH = 'path'
+
+TETHYS_COMMANDS = [MANAGE_START, MANAGE_COLLECTSTATIC, MANAGE_COLLECTWORKSPACES, MANAGE_COLLECT,
```

Comment:
Have you tested collectstatic? Seems there would be a collision with the django `manage.py collecstatic` command.

Reply:
the unknown commands are evaluated last.